cmd/clairctl/cmd/analyse.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/template"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/coreos/clair/cmd/clairctl/clair"
+	"github.com/coreos/clair/cmd/clairctl/docker"
+	"github.com/coreos/clair/cmd/clairctl/xerrors"
+)
+
+const analyseTplt = `
+Image: {{.String}}
+ {{.Layers | len}} layers found
+ {{$ia := .}}
+ {{range .Layers}} ➜ {{with .Layer}}Analysis [{{.|$ia.ShortName}}] found {{.|$ia.CountVulnerabilities}} vulnerabilities.{{end}}
+ {{end}}
+`
+
+var analyseCmd = &cobra.Command{
+	Use:   "analyse IMAGE",
+	Short: "Analyse Docker image",
+	Long:  `Analyse a Docker image with Clair, against Ubuntu, Red hat and Debian vulnerabilities databases`,
```

Comment:
Don't specify the vulnerability databases. Clair can be extended by anyone, at any time using the registration pattern.